library/common/1.0.0/templates/lib/container/_securityContext.tpl

```diff
@@ -0,0 +1,119 @@
+{{/* Returns Container Security Context */}}
+{{/* Call this template:
+{{ include "ix.v1.common.lib.container.securityContext" (dict "rootCtx" $ "objectData" $objectData) }}
+rootCtx: The root context of the chart.
+objectData: The object data to be used to render the container.
+*/}}
+{{- define "ix.v1.common.lib.container.securityContext" -}}
+  {{- $rootCtx := .rootCtx -}}
+  {{- $objectData := .objectData -}}
+
+  {{/* Initialize from the "global" options */}}
+  {{- $secContext := fromJson (include "ix.v1.common.lib.container.securityContext.calculate" (dict "rootCtx" $rootCtx "objectData" $objectData)) }}
+runAsNonRoot: {{ $secContext.runAsNonRoot }}
+runAsUser: {{ $secContext.runAsUser }}
+runAsGroup: {{ $secContext.runAsGroup }}
+readOnlyRootFilesystem: {{ $secContext.readOnlyRootFilesystem }}
+allowPrivilegeEscalation: {{ $secContext.allowPrivilegeEscalation }}
+privileged: {{ $secContext.privileged }}
+seccompProfile:
+  type: {{ $secContext.seccompProfile.type }}
+  {{- if eq $secContext.seccompProfile.type "Localhost" }}
```

Comment:
To be sure it starts with caps `L`?

Reply:
Yes (It's how k8s api docs wants it defined)